hazelcast/include/hazelcast/client/spi/impl/ClientExecutionServiceImpl.h

```diff
@@ -40,8 +40,7 @@ class HAZELCAST_API ClientExecutionServiceImpl
   : public std::enable_shared_from_this<ClientExecutionServiceImpl>
 {
 public:
-    ClientExecutionServiceImpl(const std::string& name,
-                               const client_properties& properties,
+    ClientExecutionServiceImpl(const client_properties& properties,
```

Comment:
As this is a public API, we should not change the signature. Could you revert this change?